src/components/WalletTypeButtons.tsx

```diff
@@ -0,0 +1,251 @@
+import React, { useEffect, useState } from "react";
+import Image from "next/image";
+import { BsChevronCompactDown } from "react-icons/bs";
```

Comment:
Could reorder to look nicer

Reply:
removed  useEffect, and ueState because it was not in use.